documentation/manual/src/main/asciidoc/en-US/modules/redis.asciidoc

```diff
@@ -69,6 +70,11 @@ Defines the timeout used by the driver
 when the connection to the Redis instance is initiated.
 This configuration is expressed in milliseconds.
 The default value is `5000`.
+hibernate.ogm.redis.cluster::
+Boolean flag to enable Redis Cluster mode.
+It's strongly recommended to use simple primary and association keys
```

Comment:
Can you add a new JIRA issue and point to it here should you decide to do this not as part of this PR?